ciao-storage/ceph.go

```diff
@@ -175,7 +185,7 @@ func (d CephDriver) GetVolumeMapping() (map[string][]string, error) {
 	cmd := exec.Command("rbd", args...)
 	data, err := cmd.Output()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Error when running: %v: %v: %s", cmd.Args, err, err.(*exec.ExitError).Stderr)
```

Comment:
@rbradford Sorry this is my fault, but reading the docs again the error is not guaranteed to be of type ExitError.  If it's not the cast will panic.

> Any returned error will usually be of type *ExitError. If c.Stderr was nil, Output populates ExitError.Stderr. 

So we should probably write
```
var output []byte
if err, ok := err.(*exec.ExitError); ok {
  output = err.Stderr
}
```

to be safe.